mac

```diff
@@ -180,25 +180,24 @@ install_asdf_plugin "nodejs" "https://github.com/asdf-vm/asdf-nodejs.git"
 
 install_asdf_language() {
   local language="$1"
-  local version
-  version="$(asdf list-all "$language" | tail -1)"
-
-  if ! asdf list "$language" | grep -Fq "$version"; then
-    asdf install "$language" "$version"
-    asdf global "$language" "$version"
+  local version="$2"
+  if [ -z  "$version" ]; then
+    version="$(asdf list-all "$language" | tail -1)"
   fi
+
+  asdf install "$language" "$version"
+  asdf global "$language" "$version"
 }
 
 fancy_echo "Installing latest Ruby..."
-install_asdf_language "ruby"
+install_asdf_language "ruby" "2.4.1"
 gem update --system
 gem_install_or_update "bundler"
 number_of_cores=$(sysctl -n hw.ncpu)
 bundle config --global jobs $((number_of_cores - 1))
 
 fancy_echo "Installing latest Node..."
-bash "$HOME/.asdf/plugins/nodejs/bin/import-release-team-keyring"
-install_asdf_language "nodejs"
+install_asdf_language "nodejs" "7.8.0"
```

Comment:
Can we change this to 8.7.0 ?  I just upgraded locally and it works fine.  It breaks at 9.

Reply:
I actually have a separate branch where I'm working on the upgrade. I'd like to keep the two work items separate if that's ok. Step 1, prepare the laptop script for upgrading. Step 2, do the upgrade.

Reply:
Let me know if it's cool to just merge as-is, and then do the upgrade in a later commit.